content/dart/concepts/list/terms/join/join.md

```diff
@@ -0,0 +1,108 @@
+---
+Title: 'join' 
+Description: 'The join() method concatenates (combines) list elements as a string including various separators between them dependent on a user's need.'
+Subjects:
+  - 'Code Foundations'
+  - 'Web Development'
+Tags:
+  - 'Concatenation'
+  - 'Dart'
+  - 'Lists'
+CatalogContent:
+  - 'paths/computer-science'
+---
+
+The **`join()`** method concatenates (*combines*) list elements as a string including various separators between them dependent on a user's need.
+ 
+
+## Syntax
+
+```pseudo
+list-name.join(separator); // the raw syntax
+
+chores.join(" + "); // a real example
+```
+
+The method refers to its object, in the below case, to the 'chores' variable, which holds list (or array) elements. Inside the brackets '( )' there's a parameter - a separator containing any sign/-s wrapped with single/double quotes. The main role of a separator is to separate elements from one another - allowing for better readability and additional or new meaning to the text. Spaces added to a separator are nonobligatory.

```

Comment:
As I've removed the example from the syntax, remove the description of it too. You can describe the parameters using the following format - 
```
- `parameter`:  Describe the parameter.
```

Reply:
I completely do not understand your request. Could you rephrase, please? Rather than suggesting lots of changes, is it possible to write down the reasons behind them?

Reply:
I thought that there must be also a description of syntax in general, not a specific parameter, because it would not be logical.